openc3-cosmos-init/plugins/packages/openc3-vue-common/src/components/TargetPacketItemChooser.vue

```diff
@@ -406,14 +406,22 @@ export default {
     updatePackets: function () {
       if (this.selectedTargetName === 'UNKNOWN') {
         this.packetNames = [this.UNKNOWN]
-        this.selectedPacketName = this.packetNames[0].value
+        if (this.packetNames.length === 0) {
```

Comment:
Not sure why this check is here because we set `this.packetNames = [this.UNKNOWN]` so it is always length 1